bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -220,10 +251,22 @@ h_iterations = 0
 
 orig_fore_stat = fore_stat
 
+# Assign a new variable to keep track of whether we care about fnlouder or
+# fnlouder_exc. Whether we want to remove hierarchically against inclusive
+# or exclusive background.
+if args.max_hierarchical_removal != 0:
+    # If user wants to remove against inclusive background.
+    if is_bkg_inc == True:
+        louder_foreground = fnlouder
+
+    # Otherwise user wants to remove against exclusive background
+    else :
+        louder_foreground = fnlouder_exc
+
 # Step 2 : Loop until we don't have to hierarchically remove anymore. This
 #          will happen when fnlouder has no elements that equal 0.
 
-while numpy.any(fnlouder == 0):
+while numpy.any(louder_foreground == 0):
```

Comment:
There is currently a logical error, that if you run pycbc_coinc_statmap without `--max-hierarchical-removal` this while loop will never know what `louder_foreground` is. This isn't hard to fix, it's just a matter of how cumbersome of a fix I decide to make. [Need a bit more coffee too]

Reply:
@stevereyes01 I would do the simple thing of
```python
while (args.max_hierarchical_removal > 0) and numpy.any(louder_foreground == 0):
 ```
since you immediately test
```python
    if (h_iterations == args.max_hierarchical_removal): 
        break
```
and break out of the loop if ``args.max_hierarchical_removal`` is zero anyway.